src/app/js/formats/hierarchy/Hierarchy.js

```diff
@@ -698,69 +721,26 @@ class Hierarchy extends PureComponent {
-                            fill="#010002"
-                        />
-                        <path
-                            fill="#010002"
-                            d="M114.742 70.936l-4.807 5.87h9.602zM114.913 121.223l4.796-5.869h-9.591z"
-                        />
-                    </svg>
-
-                    <ReactTooltip id="mouseIconTooltip" effect="solid">
-                        <ul>
-                            {polyglot.t('user_can_interact_with_mouse_1')}
-                            {<br />}
-                            {polyglot.t('user_can_interact_with_mouse_2')}
-                        </ul>
-                    </ReactTooltip>
+                    <MouseIcon polyglot={polyglot} />
                 </div>
 
-                {/* 2) CenterIcon */}
                 <div
                     style={{
                         position: 'absolute',
-                        bottom: '19px',
-                        left: '50px',
+                        bottom: '33px',
+                        left: '55px',
                     }}
                     onClick={this.centerGraphClick}
+                    className={stylesWithClassnames.icon}
                 >
-                    <svg
-                        data-tip
-                        data-for="centerIconTooltip"
-                        width="45"
-                        height="45"
-                        viewBox="0 0 100 125"
-                    >
-                        <g fill="#000" fillRule="evenodd">
-                            <path d="M50 59c-4.962 0-9-4.038-9-9s4.038-9 9-9 9 4.038 9 9-4.038 9-9 9m0-22c-7.18 0-13 5.82-13 13s5.82 13 13 13 13-5.82 13-13-5.82-13-13-13" />
-                            <path d="M50 75c-13.807 0-25-11.193-25-25s11.193-25 25-25c13.808 0 25 11.193 25 25S63.808 75 50 75m41.005-27H78.923C77.937 33.601 66.399 22.063 52 21.077V10.994A1.999 1.999 0 0 0 50 9c-1.112 0-2 .893-2 1.994v10.083C33.601 22.063 22.063 33.601 21.077 48H8.995A1.998 1.998 0 0 0 7 50c0 1.112.893 2 1.995 2h12.082C22.063 66.399 33.601 77.937 48 78.923v10.083c0 1.099.896 1.994 2 1.994 1.112 0 2-.893 2-1.994V78.923C66.399 77.937 77.937 66.399 78.923 52h12.082A1.998 1.998 0 0 0 93 50c0-1.112-.894-2-1.995-2" />
-                        </g>
-                    </svg>
-
-                    <ReactTooltip
-                        id="centerIconTooltip"
-                        place="right"
-                        effect="solid"
-                    >
-                        <ul>{polyglot.t('graph_reinit')}</ul>
-                    </ReactTooltip>
+                    <CenterIcon polyglot={polyglot} />
```

Comment:
No `onClick` on `CenterIcon`?

Reply:
Moved function `centerGraphClick` from `Hierarchy.js` to `CenterIcon.js`.